fast_llm/data/data/gpt/config.py

```diff
@@ -48,14 +48,15 @@ class GPTDataConfig(DataConfig, GPTLegacyConfig):
         desc="Multiprocessing context. Do not touch.",
         hint=FieldHint.expert,
     )
-    truncate_documents: bool = Field(
-        default=True,
+    truncate_documents: bool | None = Field(
```

Comment:
That works, but we normally do backward compatibility in `_from_dict`, see example in lines 73-90 below. This one can go in `GPTTrainerConfig._from_dict`.
Also needs a todo for removal.